src/lib/helpers/getFetchClient.ts

```diff
@@ -34,5 +37,39 @@ export const getFetchClient = (konfigStore: Store): {
         };
     }
 
-    return globalThis.fetch;
+    return (input: FetchInputParameter, init?: FetchInitParameterWithClient) =>
+        fetchShim(konfigStore, input, init);
 };
+
+function fetchShim(
+    konfigStore: Store,
+    input: FetchInputParameter,
+    init?: FetchInitParameterWithClient,
+): FetchReturn {
+    const fetchTimeout = konfigStore.get("networking.fetch_timeout_ms");
+    const fetchRetry = konfigStore.get("networking.fetch_retry_enable");
+    const fetchMaxAttempts = konfigStore.get("networking.fetch_retry_times");
+    const fetchInitialDebounce = konfigStore.get(
+        "networking.fetch_retry_initial_debounce",
+    );
+    const fetchDebounceMultiplier = konfigStore.get(
+        "networking.fetch_retry_debounce_multiplier",
+    );
```

Comment:
Happy to change up these config keys to fit preferred language if these aren't super clear.